#!/usr/bin/env python
PACKAGE = "local_planner"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("obj_cost",		double_t, 0, "obj_cost",    	  9.0, 0, 100);
gen.add("cur_cost",		double_t, 0, "cur_cost",    	  3.0, 0, 100);
gen.add("dst_cost",		double_t, 0, "dst_cost",    	  28.0, 0, 100);
gen.add("var_cost",		double_t, 0, "var_cost",    	  0.0, 0, 100);

gen.add("cur_diff",		double_t, 0, "cur_diff",    	  0.16, 0, 1);

exit(gen.generate(PACKAGE, "local_planner", "LocalPlanner"))
